rcl/test/rcl/test_logging_rosout.cpp

```diff
@@ -67,43 +67,39 @@ std::ostream & operator<<(
 
 class CLASSNAME (TestLogRosoutFixtureNotParam, RMW_IMPLEMENTATION) : public ::testing::Test {};
 
-class TEST_FIXTURE_P_RMW (TestLoggingRosoutFixture)
-  : public ::testing::TestWithParam<TestParameters>
+class TestLoggingRosout : public ::testing::Test
```

Comment:
just confirmation for my understanding. introducing this class as generic, so that this can be used in rosout parameter test and new test. correct?

Reply:
Yes.